src/tsolvers/egraph/UFInterpolator.h

```diff
@@ -117,7 +118,7 @@ class CGraph : public TheoryInterpolator
   void     addCEdge      ( PTRef, PTRef, PTRef );
   void removeCEdge(CEdge *);
   void     revertEdges   ( CNode * );
-  PTRef  getInterpolant( const ipartitions_t & , map<PTRef, icolor_t>*);
+  PTRef    getInterpolant( const ipartitions_t & , map<PTRef, icolor_t>*, PartitionManager&);
```

Comment:
Could TheoryInterpolator have ParititonManager as a field or something?

Reply:
I would wait with this for the refactoring of the interpolation engines.